src/components/artistas/ArtistaPortfolio.jsx

```diff
@@ -112,51 +118,67 @@ class ArtistaPortfolio extends Component {
       <section>
         <div className='grid-row margin-bottom-tiny'>
           <div className='grid-item item-s-12'>
-            <h4 className='font-size-small font-bold margin-bottom-tiny'>Portafolio</h4>
+            <h4 className='font-size-basic font-bold margin-bottom-tiny'>Portafolio</h4>
           </div>
         </div>
 
-        <div className='artista-portfilio-items'>
+        <div className='list-rows-holder margin-bottom-small'>
           {this.props.items.map( (item, index) => {
-            const { title, images, year, dimensions } = item;
+            const { title, images, year, materials, dimensions, notesRawContent } = item;
 
             let imageUrl = undefined;
 
             if(images !== undefined) {
-              imageUrl = getResizedImageUrl(images[0], '750', false);
+              imageUrl = images[0].downloadURL;
```

Comment:
why revert to non-resized image?

Reply:
because it isn't immediately available. this is @mixcas idea